docs/cloud-cloudstack.md

```diff
@@ -0,0 +1,20 @@
+### Configuration file
+
+You need to create a configuration file in INI format with your api key in `$HOME/.cloudstack.ini`
```

Comment:
I'm also thinking about how we can integrate with an already configured environment. Eg: If someone has already configured `$HOME/.cloudstack.ini` to use another provider. 
We either may specify an environment variable `CLOUDSTACK_CONFIG` or split up the config files on [regions](https://docs.ansible.com/ansible/latest/scenario_guides/guide_cloudstack.html#regions). 
I'd prefer the method with `CLOUDSTACK_CONFIG` as it more flexible.
More details might be found [here](https://docs.ansible.com/ansible/latest/scenario_guides/guide_cloudstack.html)

Reply:
`CLOUDSTACK_CONFIG` method:
- We add a prompt asking for a path to the config file
- We specify the variable for the whole block in the role ([example](https://docs.ansible.com/ansible/latest/scenario_guides/guide_cloudstack.html#environment-variables))

Reply:
made the modifications, let me know if it's what you had in mind

Reply:
It seems the default value is not specified for `_cs_region`

Reply:
> @0xmilkmix It seems you need to amend the author of your previous commits
> 
> ✅ 0xmilkmix
> ❌ milkmix

should be ok now